packages/shared/locales/en.json

```diff
@@ -805,8 +805,9 @@
             "privPolicyCheckbox": "I've read and I accept the updated Privacy Policy"
         },
         "singleAccountGuide": {
-            "title": "New: manage your funds more easily",
-            "body": "To start preparing for the future, Firefly has changed the way to manage your funds! You can now toggle between your individual wallets in the top menu bar."
+            "title": "The way you use firefly has changed",
```

Comment:
We should capitalize Firefly here (and everywhere).

Reply:
Fixed